include/mtl_api.h

```diff
@@ -516,11 +528,16 @@ struct mtl_init_params {
   /**
    * Suggest using rss (L3 or L4) for rx packets direction.
    */
-  enum mt_rss_mode rss_mode;
+  enum mtl_rss_mode rss_mode;
   /**
    * Select default or force IOVA mode.
    */
-  enum mt_iova_mode iova_mode;
+  enum mtl_iova_mode iova_mode;
+  /**
+   * Interface network protocol
+   * static or DHCP
+   */
+  enum mtl_net_proto net_proto[MTL_PORT_MAX];
```

Comment:
Can you add some comments on `sip_addr`, it need a value in no dhcp case. Also the netmask and gateway
```
  /** source IP of ports, for MTL_PMD_DPDK_USER */
  uint8_t sip_addr[MTL_PORT_MAX][MTL_IP_ADDR_LEN];
```